CMakeLists.txt

```diff
@@ -37,22 +37,27 @@ include(docs)
 add_docs_target(docs BUILDER html DEPENDS install-scipp)
 add_docs_target(doctest BUILDER doctest DEPENDS docs)
 add_docs_target(linkcheck BUILDER linkcheck DEPENDS docs)
+add_custom_target(
+  cleanup_docs_html
+  COMMENT "Remove unnecessary .ipynb files from built HTML documentation."
+  COMMAND find "${CMAKE_BINARY_DIR}/html" -type f -name "*.ipynb" -not -path
+          "${CMAKE_BINARY_DIR}/html/_sources/*" -delete
+  VERBATIM
+)
+add_dependencies(cleanup_docs_html docs)
```

Comment:
This could trigger an accidental build of `docs` if someone calls `cleanup_docs_html` without building the docs. (May happen in the future when refactoring.) I don't think we need a dependency here?

Reply:
Not sure. Is there a guarantee about the order cmake builds targets if a build preset specifies multiple targets?